resources/lua_styles/filter-highway.lua

```diff
@@ -277,3 +274,15 @@ function osm2pgsql.process_relation(object)
         members = object.members,
     })
 end
+
+-- Process tagged objects:
+osm2pgsql.process_node = do_nodes
+osm2pgsql.process_way = do_ways
+osm2pgsql.process_relation = do_relations
+
+-- If osm2pgsql is of version `2.0.0`, assign process_untagged_* functions:
+if osm2pgsql.version == '2.0.0' then
```

Comment:
Probably better to let it be more general, probably also all minor versions 2.*.*

Reply:
New function was implemented in helper.lua module that will handle cases of `osm2pgsql` versions being equal to or greater than `2.0.0.` as acceptable. I assume the way untagged objects are handled will stay unchanged until the next major update